code/modules/mentor/verbs/mentorhelp.dm

```diff
@@ -0,0 +1,106 @@
+/client/verb/mentorhelp(msg as text)
+	set category = "Mentor"
+	set name = "Mentorhelp"
+
+	//clean the input msg
+	if(!msg)
+		return
+	msg = sanitize(copytext(msg,1,MAX_MESSAGE_LEN))
+	if(!msg || !mob)
+		return
+	if(src.prefs.muted & MUTE_MENTORHELP)
+		to_chat(src, "<font color='red'>You are unable to use mentorhelp (muted).</font>")
+		return
+	var/show_char = CONFIG_GET(flag/mentors_mobname_only)
+	var/mentor_msg = "<span class='mentornotice'><b><font color='#3280ff'>MENTORHELP:</b> <b>[key_name_mentor(src, 1, 0, 1, show_char)]</b>:</font> [msg]</span>"
+	log_mentor("MENTORHELP: [key_name_mentor(src, 0, 0, 0, 0)]: [msg]")
+
+	for(var/client/X in GLOB.mentors)
+		X << 'sound/items/bikehorn.ogg'
+		to_chat(X, mentor_msg)
+
+	for(var/client/A in GLOB.admins)
+		A << 'sound/items/bikehorn.ogg'
+		to_chat(A, mentor_msg)
+
+	to_chat(src, "<span class='mentornotice'><font color='purple'>PM to-<b>Mentors</b>: [msg]</font></span>")
```

Comment:
You use the span class `mentornotice` twice, but it's not defined. Make sure it's in both `interface/stylesheet.dm` and `code/modules/goonchat/browserassets/css/browserOutput.css`